packages/main/src/Link.ts

```diff
@@ -167,11 +167,11 @@ class Link extends UI5Element implements ITabbable {
 	/**
 	 * Defines how the text of a component will be displayed when there is not enough space.
 	 *
-	 * **Note:** for option "Normal" the text will wrap and the words will not be broken based on hyphenation.
-	 * @default "None"
+	 * **Note:** for option "None" the text won't wrap and the words will be broken based on hyphenation.
```

Comment:
To me the note 
`for option 'None' the text won't wrap and the words will be broken based on hyphenation`
is confusing  as in first part we say `text won't wrap` and then in the scond part - `will be broken` based on the hyphanation.


can't we just say:

```
**Note:** By default the text will wrap. If "None" is set - the text will truncate. 
```